Client/Frontend/Fakespot/FakespotViewModel.swift

```diff
@@ -12,7 +12,7 @@ class FakespotViewModel {
         case loaded(ProductAnalysisData?)
         case error(Error)
 
-        var viewElements: [ViewElement] {
+        fileprivate var viewElements: [ViewElement] {
```

Comment:
Why `fileprivate` and not `private`?

Reply:
If it's private `FakespotViewModel` can't access it.

Reply:
You mean `FakespotViewController?` Anyway, not important you can merge the PR!

Reply:
Looks like there's 2 viewElements, one in the state and one in view model, the one in the view model wants to be hidden as private, but if it's private it cannot be accesed in view model, so that's why I think it's fileprivate.